core/templates/pages/exploration-editor-page/changes-in-human-readable-form/changes-in-human-readable-form.component.html

```diff
@@ -49,70 +57,18 @@
             </span>
           </div>
 
-          <div *ngSwitchCase="'answer_groups'">
-            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'added'" class="state-edit-desc answer-group">
-              <strong>Added answer group: </strong>
-              <p class="sub-edit">
-                <i>Destination: </i>{{ lostChange.newValue.outcome.dest }}
-              </p>
-
-              <div class="sub-edit">
-                <i>Feedback: </i>
-                <div class="feedback">
-                  {{ lostChange.newValue.outcome.feedback.html }}
-                </div>
-              </div>
-
-              <div class="sub-edit" *ngIf="lostChange.newValue.rules.length > 0">
-                <i>Rules: </i>
-                <ol class="rules-list">
-                  <li *ngFor="let rule of lostChange.newValue.rules">
-                    <p>Type: {{ rule.type }}</p>
-                    <p>Value:
-                      <span> </span>
-                      <ng-container *ngFor="let input of rule.inputs | keyvalue; let last = last">
-                        {{ last ? input.value: (input.value + ',') }}
-                      </ng-container>
-                    </p>
-                  </li>
-                </ol>
-              </div>
-
-            </div>
-
-            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'edited'" class="state-edit-desc answer-group">
-              <strong>Edited answer group: </strong>
-              <p class="sub-edit" *ngIf="!lostChange.isOutcomeDestEqual()">
-                <i>Destination: </i>
-                {{ lostChange.newValue.outcome.dest }}
-              </p>
-
-              <div class="sub-edit" *ngIf="lostChange.isOutcomeFeedbackEqual()">
-                <i>Feedback: </i>
-                <div class="feedback">
-                  {{ lostChange.newValue.outcome.feedback.html }}
-                </div>
-              </div>
+          <div *ngSwitchCase="'answer_groups'"  class="state-edit-desc">
+            <span *ngIf="lostChange.getRelativeChangeToGroups() === 'added'">
+              Added answer group
+            </span>
 
-              <div class="sub-edit" *ngIf="lostChange.isRulesEqual()">
-                <i>Rules: </i>
-                <ol class="rules-list">
-                  <li *ngFor="let rule of lostChange.newValue.rules; trackBy: index">
-                    <p>Type: {{ rule.type }}</p>
-                    <p>Value:
-                      <span> </span>
-                      <ng-container *ngFor="let input of rule.inputs | keyvalue; let last = last">
-                        {{ last ? input.value: (input.value + ',') }}
-                      </ng-container>
-                    </p>
-                  </li>
-                </ol>
```

Comment:
Ditto: losing lot of info compared to previous version implementation.

Reply:
Ditto same as above.